Resources/Prototypes/Maps/cog.yml

```diff
@@ -3,7 +3,6 @@
   mapName: 'Cog'
   mapPath: /Maps/cog.yml
   minPlayers: 50
-  maxPlayers: 80 #big map
```

Comment:
in the unlikely event we increase our server cap, we will still want this capped. cog is big, but its no fland